src/main/arguments/InputParser.groovy

```diff
@@ -20,16 +21,17 @@ class InputParser {
 
             if (lineMap.containsKey("name")) {
                 String name = line["name"]
-                
+
                 projectList.add(new Project(name, path))
             } else {
                 projectList.add(new Project(path))
             }
         }
-
+        if(projectList.size() == 0) throw new InvalidArgsException('The input file cannot be processed, the project list will be considered as empty')
```

Comment:
This exception stops execution right? In this case, I don't think the second part of the message is needed. Could be just 

```java
throw new InvalidArgsException('The input file cannot be processed')
```